Libraries/PeriphDrivers/Source/SPIMSS/spimss_reva.c

```diff
@@ -231,6 +242,197 @@ int MXC_SPIMSS_RevA_MasterTrans(mxc_spimss_reva_regs_t *spi, spimss_reva_req_t *
     return E_NO_ERROR;
 }
 
+/* ************************************************************************** */
+void MXC_SPIMSS_RevA_DMA_Handler(int ch, int error)
+{
+    int transaction_size = 0;
+    spimss_reva_req_t *temp_req = NULL;
+
+    for (int i = 0; i < MXC_SPIMSS_INSTANCES; i++) {
+        temp_req = states[i].req;
+        transaction_size = (temp_req->bits / 8) * (temp_req->len);
+
+        if (ch == states[i].channelTx) {
+            states[i].req->tx_num = transaction_size - (MXC_DMA)->ch[ch].cnt;
+
+            if ((true == states[i].auto_dma_handlers) && (0 == (MXC_DMA)->ch[ch].cnt)) {
+                //Disable interrupts.
+                MXC_DMA_DisableInt(ch);
+                MXC_DMA_SetChannelInterruptEn(ch, false, false); // Disable ctz interrupt.
+                MXC_DMA_ReleaseChannel(ch);
+                states[i].spi->dma &= ~(1 << 15); // Disable SPIMSS TX DMA requests
```

Comment:
Instead of using hard-coded values, please use the register definitions from [spimss_regs.h](https://github.com/Analog-Devices-MSDK/msdk/blob/main/Libraries/CMSIS/Device/Maxim/MAX32660/Include/spimss_regs.h)

For example, this should be
```C
states[i].spi->dma &= ~(MXC_F_SPIMSS_DMA_TX_DMA_EN);
```

Reply:
Updated